kaiax/gasless/impl/builder_test.go

```diff
@@ -55,15 +55,17 @@ func TestExtractTxBundles(t *testing.T) {
 
 	S3 := makeSwapTx(t, nil, 0, SwapArgs{Token: common.HexToAddress("0xabcd"), AmountIn: big.NewInt(10), MinAmountOut: big.NewInt(100), AmountRepay: big.NewInt(1021000)})
 
-	T1 := makeTx(t, nil, 0, common.HexToAddress("0xAAAA"), big.NewInt(0), 1000000, big.NewInt(0), nil)
-	T2 := makeTx(t, nil, 0, common.HexToAddress("0xAAAA"), big.NewInt(0), 1000000, big.NewInt(0), nil)
+	T4 := makeTx(t, nil, 0, common.HexToAddress("0xAAAA"), big.NewInt(0), 1000000, big.NewInt(1), nil)
+	T5 := makeTx(t, nil, 0, common.HexToAddress("0xAAAA"), big.NewInt(0), 1000000, big.NewInt(1), nil)
```

Comment:
What do these numbers mean?

Reply:
Sender number. For example, A1 and S1 are sender1's txs, T4 is sender4's tx.

Reply:
I see, could you write it as comment please?

Reply:
fixed at https://github.com/kaiachain/kaia/pull/280/commits/9427630bad8d4a04a8696945f433cb1d560ee73b